Userland/DevTools/HackStudio/Git/GitWidget.cpp

```diff
@@ -146,6 +153,11 @@ void GitWidget::set_view_diff_callback(ViewDiffCallback callback)
     m_view_diff_callback = move(callback);
 }
 
+void GitWidget::set_repository_update_callback(Function<void()> callback)
```

Comment:
Do not use `set_` for setters.
```suggestion
void GitWidget::repository_update_callback(Function<void()> callback)
```

Reply:
Sorry for going back and forth on this, but please _do_ use the `set_` prefix here as per the coding style guidelines : https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#right-3

>  Precede setters with the word "set". Use bare words for getters. Setter and getter names should match the names of the variables being set/gotten.